template/python3-flask/function/handler_test.py

```diff
@@ -0,0 +1,8 @@
+from .handler import handle
+
+# Test your handler here
+# To disable testing, you need to update the
+# tox.ini file, in addition to deleting this file.
+
+def test_handle():
+    assert handle("input") == "input"
```

Comment:
What if we made this `pass` instead? So that if there's an issue with layers or backwards compatibility, it always works?